.pre-commit-hooks.yaml

```diff
@@ -3,7 +3,7 @@
     description: ''
     entry: clang-format -i
     language: python
-    'types_or': [c++, c, c#, cuda, java, javascript, json, objective-c, proto]
+    'types_or': [c++, c, c#, cuda, java, javascript, json, objective-c, proto, textproto]
```

Comment:
this is generated

Reply:
Huh? By what? But OK, I've reverted it...